docs/basics/part-8a-using-earthly-in-your-current-ci.md

```diff
@@ -11,10 +11,17 @@ For more information on how to use Earthly in other CIs such as GitLab, Jenkins,
 To use Earthly in a CI, you typically encode the following steps in your CI's build configuration:
 
 1. Download and install Earthly
-2. Log in to image registries, such as DockerHub
-3. Run Earthly
+2. Set up any credentials needed for the build
+3. Log in to image registries, such as DockerHub
+4. Run Earthly
 
-For example, here is how this would work in GitHub Actions:
+As part of this, you may need to set up credentials for Earthly itself, if you are using Earthly Satellites, or Earthly Cloud Secrets. For this, you can use the following command:
+
+```bash
+earthly account create-token my-ci-token
```

Comment:
to avoid expiry?

```suggestion
earthly account create-token --expiry=never my-ci-token
```

Reply:
I believe that we're making "never" the default. Right @idodod?

Reply:
yeah, I didn't look closely into earthly core side but @mikejholly already pushed the change in the BE that changed the default to never.

Reply:
I'm backed up a little but can take probably later today or early tomorrow.

Reply:
SGTM